internal/data/config.go

```diff
@@ -201,13 +201,24 @@ func (d *database) GetWebauthn() (wba Webauthn, err error) {
 	}
 
 	wba.Origin = tunnelURL
```

Comment:
Since you're not changing the Origin, doesnt this still break for your port issues?

Reply:
Could you have a look at this comment too?